### DISCUSSION TITLE
cant load file from nfs

### DISCUSSION THREAD

**Discussion by chasingw** [foxglove/community#413] [Q&A] [answered]
I mount file by nfs in ubuntu 18.04 use command like `sudo mount -t nfs xxxx:xxx xxxx`. I can use the file in other software like files, brower. but I cant see the file in foxglove.

**Comment by jtbandes** [comment]
Do you get any errors in the app?

**Comment by chasingw** [comment]
> Do you get any errors in the app?

no errors. when I drop a file to the studio, it display `drop a file here` and nothing happened. if I click Open local file and I cant see the nfs mounted directory in the file dialog

**Comment by jtbandes** [comment]
Any errors in the developer console?

**Comment by chasingw** [comment]
> Any errors in the developer console?

sorry, I use the desktop app in ubuntu 18.04, how to show the developer console

**Comment by jtbandes** [comment]
Alt > View > Advanced

**Comment by chasingw** [comment]
> Alt > View > Advanced

thanks and the console log when drop a nfs file to the studio
```
Workspace.tsx:403 Uncaught (in promise) DOMException: A requested file or directory could not be found at the time an operation was processed.
(anonymous) @ Workspace.tsx:403
await in (anonymous) (async)
(anonymous) @ Workspace.tsx:475
(anonymous) @ DocumentDropListener.tsx:141
```

**Comment by chasingw** [comment, marked as answer]
error in snap version, fix by install deb

**Reply by jtbandes** [reply]
Maybe some kind of permission issue?
